### ISSUE TITLE
Suggestion on panel icons size

### ISSUE THREAD

**Issue by q4osteam** [seb3773/q4osXpack#22]
We have tinkered on Win10 theme a little bit. You may considered a small change of size of main panel icons, what may give a better user experience. At default the icons are set to small size, however they could be configured little bit bigger, see attachments.
![s1](https://github.com/seb3773/q4osXpack/assets/16801039/a3d631bb-be12-4d54-8375-df3707537ef0)
![s2](https://github.com/seb3773/q4osXpack/assets/16801039/0ed20dad-cf2d-44d3-acd7-6b8ce8491972)
![s3](https://github.com/seb3773/q4osXpack/assets/16801039/46c0f729-485b-418e-a38f-eb3d0fffee5a)

**Comment by seb3773** [comment]
You're right, the user experience is a bit better with little bigger icons; although the actual proportions "taskbar height / icons size) is really close to the original windows 10 aspect. I already did some test about it: When I increase the taskbar icons size, I have to increase too the taskbar "apps launcher" (launcher_panelapplet_modernui_rc) & systray icon size too to avoid a disproportion that is not very elegant, did you not have this problem ? (well, it is not really a problem though, I just have to adjust the script to the desired values).

I think one the reasons the taskbar is more user friendly with bigger icons is the "active windows" indicator too, which is not present when you select icons below a certain size (30 pixels, something like that I think, need to take a look); which is something I wanted to ask you for: can we have this indicator too with litlle size icons ? I see no reasons why it's not here below a certain size (apart maybe "technical" reasons ?)

Anyway, I can add an option when you select the theme, to choose between "original windows taskbar icons size" or "bigger taskbar icons  size", to give the choice to the user, what do you think ?

**Comment by q4osteam** [comment]
Understand, so we would prefer not to complicate things and let it as is at the moment. A cleaner solution would be to modify TDE panel and applets code to draw icons more consistent and proportional way.

> .. can we have this indicator too with litlle size icons ?

Yes, code modifications are needed.

Now you can close this report and possibly create a new one, with a proper subject and a request of modifications for panel applets.
